README.md

```diff
@@ -1,54 +1,11 @@
-![cf](https://i.imgur.com/7v5ASc8.png) lab-07-cowsay-http-server
-======
+Project 07 401
```

Comment:
Nice job including your readme.  To expand I would also include information on how to actually use your routes.